### DISCUSSION TITLE
GitHub Actions Not Run on PR Generated by GitHub Actions

### DISCUSSION THREAD

**Discussion by osterman** [community/community#25602] [Actions]
<p>I’ve found a dozen or more posts talking about GitHub Actions not running on forks or other kinds of PRs. That’s not my problem. </p>
<p>TL;DR: PR created by github-actions bot  <strong>does not</strong> trigger PR workflows. PR created by a human <strong>does</strong> trigger PR workflows.</p>
<ol>
<li>
<p>I have an auto-update workflow: <a href="https://github.com/cloudposse/packages/tree/master/.github/workflows" rel="nofollow noopener">https://github.com/cloudposse/packages/tree/master/.github/workflows</a> which updates the packages for a repository</p>
</li>
<li>
<p>It creates a new PR if there were any changes (in a local branch)</p>
</li>
<li>
<p>It works and creates a PR like this one: <a href="https://github.com/cloudposse/packages/pull/263" rel="nofollow noopener">https://github.com/cloudposse/packages/pull/263</a></p>
</li>
<li>
<p>But the workflows for labeling the PR and auto-assigning the PR are not run</p>
</li>
</ol>
<p>Should this be this possible? =)</p>
<p>Thanks!</p>

**Comment by peter-evans** [comment, marked as answer]
<p>Looks like you are using the <a href="https://github.com/peter-evans/create-pull-request" rel="nofollow noopener">create-pull-request</a> action I created. If you check the README there is a note about this.</p>
<blockquote>
<p><strong>Note</strong> : If you want pull requests created by this action to trigger an on: pull_request workflow then you must use a <a href="https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line" rel="nofollow noopener">Personal Access Token</a> instead of the default GITHUB_TOKEN. See <a href="https://github.com/peter-evans/create-pull-request/issues/48" rel="nofollow noopener">this issue</a> for further details.</p>
</blockquote>

**Reply by raja-anbazhagan** [reply]
is there are the way to force the job to retrigger again after an edit or something to that PR?

**Reply by florianmutter** [reply]
You can close and reopen it to trigger the actions

**Comment by osterman** [comment]
<p>Thanks, Peter! Yes, using your action (forked for security). I missed that note. Good looking out. </p>
